src/components/pages/datasets/person/utils.js

```diff
@@ -9,14 +9,16 @@ export const checkSource = (data) => {
 };
 
 export const getSourceUrl = (data, person) => {
+  let sanctionId;
   switch (data.source) {
     case Object.keys(SOURCE)[0]:
       return `${SOURCE_URL[data.source]}${person.pep_data[0].id}`;
     case Object.keys(SOURCE)[1]:
+      sanctionId = person.sanction_data.find(
+        (item) => item.decree.includes(data.year),
+      );
       return `${SOURCE_URL[data.source]}${
-        data.year ? person.sanction_data.find(
-          (item) => item.decree.includes(data.year),
-        ).id : person.sanction_data[0].id
+        sanctionId ? sanctionId.id : person.sanction_data[0].id
```

Comment:
Add link for source sanction for taxpair number by sanction_id

Reply:
done